modules/web/src/app/dynamic/enterprise/quotas/component.ts

```diff
@@ -41,6 +42,12 @@ enum Column {
   Actions = 'Actions',
 }
 
+class progressBarData {
```

Comment:
`ProgressBarData

If is used for static type inference use Interface instead of class. Although it serve the purpose but class act as a blueprint for objects. IMHO, we can use interface. thoughts?